indico/modules/events/layout/templates/page.html

```diff
@@ -3,6 +3,15 @@
 {% block title %}
     {# The title of a user-generated page isn't translated, localized title is not needed #}
     {{ page.menu_entry.title }}
+    {% if event.can_manage(session.user) %}
+    <a href="#"
```

Comment:
You should fix the indentation of this whole block ;)

Reply:
also of the attributes - check other templates, we align the attributes like this:

```
<a href="..."
   data-foo="bar">
```
```
<span class="..."
      whatever="foobar">
```